modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/ProjectGenerator.java

```diff
@@ -190,14 +190,13 @@ private static File _getTemplateFile(
 			}
 		}
 
-		File archetypesFile = FileUtil.getJarFile(ProjectTemplates.class);
-
 		String artifactId =
 			ProjectTemplates.TEMPLATE_BUNDLE_PREFIX +
 				template.replace('-', '.');
 
-		return ProjectTemplatesUtil.getArchetypeFile(
-			artifactId, archetypesFile);
+		File archetypesFile = ProjectTemplatesUtil.getArchetypeFile(artifactId);
```

Comment:
just do a `return ProjecttemplateUtil.getArchetype...` no need to store in a var just to return it.